test/search/test_organized_index.cpp

```diff
@@ -576,9 +575,9 @@ TEST (PCL, Organized_Neighbor_Search_Pointcloud_Neighbours_Within_Radius_Search_
     current = cloudSearchBruteforce.begin();
     while (current != cloudSearchBruteforce.end())
     {
-      double pointDist = sqrt ((cloudIn->points[i].x - searchPoint.x) * (cloudIn->points[i].x - searchPoint.x) +
-                               (cloudIn->points[i].y - searchPoint.y) * (cloudIn->points[i].y - searchPoint.y) +
-                               (cloudIn->points[i].z - searchPoint.z) * (cloudIn->points[i].z - searchPoint.z));
+      double pointDist = sqrt ((cloudIn->points[*current].x - searchPoint.x) * (cloudIn->points[*current].x - searchPoint.x) +
+                               (cloudIn->points[*current].y - searchPoint.y) * (cloudIn->points[*current].y - searchPoint.y) +
+                               (cloudIn->points[*current].z - searchPoint.z) * (cloudIn->points[*current].z - searchPoint.z));
```

Comment:
```suggestion
      const auto pointDist = geometry::distance(cloudIn->points[*current], searchPoint);
```

Since we're touching these lines

Reply:
Updated and also switched loops to iterators.